puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-x-develop-release.groovy

```diff
@@ -10,18 +10,23 @@ pipeline {
         ansiColor('xterm')
     }
 
-    environment {
-        ruby_version = "2.5"
-    }
-
     stages {
         stage('Copy Source') {
             steps {
                 script {
-                    sourcefile_paths = pwd(tmp: true)
+                    artifact_path = "${pwd()}/artifacts"
                     source_project_name = "${project_name}-${git_ref}-source-release"
-                    copyArtifacts(projectName: source_project_name, target: sourcefile_paths, selector: {lastSuccessful: true})
+                    copyArtifacts(projectName: source_project_name, target: artifact_path)
+                    sourcefile_paths = list_files("${artifact_path}/*").findAll { it.endsWith('.gem') || it.endsWith('tar.bz2') || it.endsWith('tar.gz') }
+                }
+            }
+        }
+        stage('Setup Environment') {
+            steps {
+                dir('foreman-packaging') {
```

Comment:
Isn't this now equivalent to `setup_nightly_build_environment`?

Reply:
Yes, but I meant to remove that function since it doesn't add much value now.